joint_trajectory_controller/CMakeLists.txt

```diff
@@ -95,6 +95,26 @@ if(BUILD_TESTING)
     controller_manager
     test_robot_hardware
   )
+
+  ament_add_gtest(
+    test_trajectory_actions
+    test/test_trajectory_actions.cpp
+  )
+  target_include_directories(test_trajectory_actions PRIVATE include)
+  target_link_libraries(test_trajectory_actions
+    joint_trajectory_controller
+  )
+  ament_target_dependencies(test_trajectory_actions
+    "controller_parameter_server"
```

Comment:
Please remove quotes from here, we've just done that for all of `ros2_control` https://github.com/ros-controls/ros2_control/pull/71 and `ros2_controllers` https://github.com/ros-controls/ros2_controllers/pull/32